packages/mosip-api/src/esignet-api.ts

```diff
@@ -201,6 +184,11 @@ const pickUserInfo = async (userInfo: OIDPUserInfo) => {
     firstName: names?.[0],
     familyName: names?.[names?.length - 1],
     middleName: names && names?.length > 2 ? names?.[1] : "",
```

Comment:
Maybe leave these here for now for backwards compability? Not sure if useful for anyone really.

Reply:
Yep, makes sense